source/documentation/deploying_apps/deploying_java.md

```diff
@@ -106,11 +106,15 @@ For more configuration options see [Tomcat Container](https://github.com/cloudfo
 
 ### Specify a Java version
 
-You should tell Cloud Foundry which version of Java your app uses in the Java buildpack.
+You can specify the version of the Java Runtime Environment (JRE) by setting the environment variable `JBP_CONFIG_OPEN_JDK_JRE`. For example to use JRE 11 you could add the following to your application's `manifest.yml`:
```

Comment:
Is this the only way to do this?

if so, suggest the following: "Specify the version of the Java Runtime Environment (JRE) by setting the environment variable `JBP_CONFIG_OPEN_JDK_JRE` in the app's `manifest.yml`. For example, to use JRE 11:

Reply:
You could use `cf set-env ...` instead of the manifest if you wanted to, but in this case the manifest is a much more sane approach I think. Might be ok just to document the manifest as the only way, even though there's technically another approach.

Could link out to [#environment-variables](https://docs.cloud.service.gov.uk/deploying_apps.html#environment-variables) though.

Reply:
I took this example style from the tomcat one in https://github.com/alphagov/paas-tech-docs/blob/e2958ca162277f99952146b8f00511d56cd9bde0/source/documentation/deploying_apps/deploying_java.md#deploying-a-war-file it does say *for example* so I would say that implies it is not the only approach but I have found it is the most helpful for example when helping people on Slack. Not precious about this if you want to change it though.

Reply:
How about a compromise: "You can specify the version of the Java Runtime Environment (JRE) by setting the environment variable JBP_CONFIG_OPEN_JDK_JRE in the app's manifest.yml. For example, to use JRE 11"?